_1327/settings.py

```diff
@@ -78,6 +78,7 @@
 	'django.contrib.sessions',
 	'django.contrib.messages',
 	'django.contrib.staticfiles',
+	'django.contrib.postgres',
```

Comment:
this is likely the reason why the tests fail.

it's needed for using the postgres full text search functionality (see [here](https://docs.djangoproject.com/en/2.0/topics/db/search/) and [here](https://docs.djangoproject.com/en/2.0/ref/contrib/postgres/search/), but that's not used right now in this PR as far as i can see.

by using that you would gain stuff like searching for multiple words which would then not need to be adjacent, and weighting the results by some relevancy metric, so it might be worth checking out?

Reply:
yes, i would suggest that for #212